src/components/expandable_section.rs

```diff
@@ -129,21 +132,47 @@ pub fn expandable_section(props: &ExpandableSectionProperties) -> Html {
         class.extend(classes!("pf-m-expanded"));
     }
 
+    let content = html!(
+        <div
+            class="pf-v5-c-expandable-section__content" hidden={!expanded}
+        >{ props.children.clone() }</div>
+    );
+
+    let truncating = matches!(props.variant, ExpandableSectionVariant::Truncate);
+    let toggle = match props.detached {
+        true => html!(),
+        false => html!(
+            <ExpandableSectionToggle
+                {ontoggle}
+                expanded={expanded}
+                toggle_text_hidden={&props.toggle_text_hidden}
+                toggle_text_expanded={&props.toggle_text_expanded}
+                detached=false
+                direction={ExpandableSectionToggleDirection::Down}
+                no_icon={truncating}
+            />
+        ),
+    };
+
+    let content = match truncating {
```

Comment:
Needs comment that this is a design choice.

Reply:
That's actually what patternfly does:

![image](https://github.com/patternfly-yew/patternfly-yew/assets/202474/ca839a97-8ec4-45e6-9116-9765f2461a6f)

I think that was a bug before.

Reply:
Yes, I understand. I just meant a comment pointing out that it's a design choice of PF because the code by itself without context looks confusing.

Reply:
Ah ok!

Reply:
Should be fixed.